ape_alchemy/provider.py

```diff
@@ -276,6 +277,16 @@ def create_access_list(
     def make_request(self, rpc: str, parameters: Optional[Iterable] = None) -> Any:
         rate_limit = self.config.rate_limit
         parameters = parameters or []
+
+        def checker(err: Exception) -> bool:
```

Comment:
I would define this outside of this method, else I think Python has to redefine the method every time `make_request()` is called

Reply:
e832333